scripts/db/src/get-user.ts

```diff
@@ -26,44 +26,68 @@ async function getByEmail(email: string, callback: DbScriptCallback) {
   //     callback(new Error("my error message"));
   try {
     /** Get required dependencies */
-    const { MongoClient } = require('mongodb@4.1.0')
+    const { Client: PGClient } = require('pg@8.7.1')
 
-    const { MONGO_URI, MONGO_DB_NAME } = configuration
+    const {
+      POSTGRES_USERNAME,
+      POSTGRES_PASSWORD,
+      POSTGRES_HOST,
+      POSTGRES_DATABASE,
+      POSTGRES_PORT,
+    } = configuration
 
     /**
      * Logic in this function tries to match:
      * EAForum/packages/lesswrong/server/vulcan-lib/apollo-server/authentication.tsx
      */
     async function getForumUser(): Promise<CallbackUser | null> {
-      // Connect to the Forum DB
-      const mongoClient: MongoClient = new MongoClient(MONGO_URI)
-      await mongoClient.connect()
+      const pgConnectionInfo: PGConnectionConfig = {
+        user: POSTGRES_USERNAME,
+        password: POSTGRES_PASSWORD,
+        host: POSTGRES_HOST,
+        database: POSTGRES_DATABASE,
+        port: POSTGRES_PORT ? parseInt(POSTGRES_PORT) : 5432,
+        ssl: TEMPLATE_DATA.pgShouldSsl,
+      }
+
+      /**
+       * NOTE: Temporary fix for Auth0 bug August 2022
+       * Should be reverted ASAP
+       *
+       * Update Nov 2022: Auth0 told me the bug is fixed so I tried
+       * reverting and it broke - the bug is very much not fixed :(
+       */
+      process.env.NODE_TLS_REJECT_UNAUTHORIZED = '0'
```

Comment:
This was here in the version that connected to parfit. We're planning to remove this script entirely soon and do the legacy authentication directly in the codebase, so I didn't check if we can remove this